ansible/provision.yaml

```diff
@@ -30,6 +30,7 @@
       when: ansible_os_family == "Flatcar"
   roles:
     - role: offline
+    - role: setup_versionlock
```

Comment:
Do we need both this and https://github.com/mesosphere/konvoy-image-builder/search?q=setup_versionlock?

Reply:
honestly im not sure. let me run this again and figure it out.